src/Factory.php

```diff
@@ -144,7 +144,11 @@ public function getComposerCommand(): ComposerCommand {
 
     public function getStatusCommand(): StatusCommand {
         return new StatusCommand(
-            $this->getPhiveXmlConfig($this->request->getOptions()->hasOption('global')),
```

Comment:
Is there a reason you removed the global option and hard-coded it?

Reply:
Nope!
My bad, it's a leftover of a previous test. I'll correct it.